synapse/push/bulk_push_rule_evaluator.py

```diff
@@ -186,6 +230,19 @@ async def action_for_event_by_user(self, event, context) -> None:
                         actions_by_user[uid] = actions
                     break
 
+            # If the event should be counted as unread, add mark_unread to its actions.
```

Comment:
I don't feel like this is a change to what is already explained in the docstring for `_get_rules_for_event`. If anything the changes fixed the function's compliance to its own docstring, since it doesn't mention any filtering:

> This gets the rules for all users in the room at the time of the event, as well as the push rules for the invitee if the event is an invite.

Reply:
fair.